src/NLog/LayoutRenderers/LevelLayoutRenderer.cs

```diff
@@ -46,13 +48,44 @@ namespace NLog.LayoutRenderers
     public class LevelLayoutRenderer : LayoutRenderer
     {
         /// <summary>
+        /// A value indicating when the cache is cleared.
+        /// </summary>
+        [Flags]
+        public enum LevelOutputOption
+        {
+            /// <summary>Ouput the name as the level.</summary>
```

Comment:
Proposal: 
- render the full level name
- render the first character for the level
- render the ordinal (aka number) for the level.